Userland/Libraries/LibJS/Runtime/AbstractOperations.cpp

```diff
@@ -205,8 +205,8 @@ ThrowCompletionOr<void> initialize_bound_name(VM& vm, FlyString const& name, Val
 {
     // 1. If environment is not undefined, then
     if (environment) {
-        // a. Perform ! environment.InitializeBinding(name, value).
-        MUST(environment->initialize_binding(vm, name, value));
+        // a. Perform ! environment.InitializeBinding(name, value, normal).
```

Comment:
Maybe add a link to the proposal in L203 here and in the ones affected below too?